src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuildingStructureBuilder.java

```diff
@@ -79,6 +97,11 @@
      */
     private int progressCounter = 0;
 
+    /**
+     * all the fluids to be removed in fluids_remove.
+     */
+    private Map<Integer, List<BlockPos>> fluidsToRemove = new TreeMap<>();
```

Comment:
this should be a ordered list.

Reply:
earlier you told me to make it an ordered map, however i forgot that and made it a tree map.
what should i make it, ordered map, or list?

Reply:
what type of map/list should this be?
java has neither a OrderedMap nor a OrderedList.

Reply:
LinkedHashMap

Reply:
done.